nglview/utils/widget_utils.py

```diff
@@ -90,3 +92,66 @@ def func(this, **kwargs):
             func = make_func(rep)
             fn = '_'.join((root_fn, rep[0]))
             setattr(self, fn, MethodType(func, other))
+
+
+def compare_two_images(image_path_1,
```

Comment:
may be use python type's annotation? https://docs.python.org/3/library/typing.html

Reply:
anh may be just "compare_images"

Reply:
> may be just "compare_images", not "compare_two_images"

this one too.